docs/site/docs/develop/faq.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 6
+---
+
+# FAQ
+
+## Developing Smart Contracts
+
+**How fast is a message delivered?**
+
+We are still collecting data on this and it depends on various parameters, but rollup to rollup messages are roughly 5-10s for "latest" messages and 12 minutes for "finalized" messages.
```

Comment:
Generally it's preferable to avoid time-related wordings so it stays up to date for as long as needed. 
"still collecting data" raises more questions, like when will we have the data, ..

This is enough IMO
"Rollup to rollup messages are roughly 5-10s for "latest" messages and 12 minutes for "finalized" messages."